src/_P078_Eastron.ino

```diff
@@ -173,229 +187,812 @@ boolean Plugin_078(uint8_t function, struct EventStruct *event, String& string)
     }
 
     case PLUGIN_WEBFORM_LOAD:
+    {
       {
-        {
-          const __FlashStringHelper * options_model[4] = { F("SDM120C"), F("SDM220T"), F("SDM230"), F("SDM630") };
-          addFormSelector(F("Model Type"), P078_MODEL_LABEL, 4, options_model, nullptr, P078_MODEL );
+        const __FlashStringHelper *options_model[7] =
+        { F("SDM120C"), F("SDM220T"), F("SDM230"), F("SDM630"), F("SDM72D"), F("DDM18SD"), F("SDM72D V2") };
+        addFormSelector(F("Model Type"), P078_MODEL_LABEL, 7, options_model, nullptr, P078_MODEL);
+        addFormNote(F("Submit after changing the modell to update Output Configuration."));
+        // ugly conversation logic to be backwards compatible
```

Comment:
Why making this conversion table?
The order of the existing models has not changed, so their enum values are still the same.
If you want to re-order the list in the combo box, you can also add an array of enum values to make sure the same nr will fit the same description.

Changing settings like this is very error prone and will render existing settings incompatible with older versions.
If such incompatibility is really the only option, then so be it, but I don't see why it has to be done like this here.